src/main/java/com/google/songgame/servlets/ChatServlet.java

```diff
@@ -122,37 +123,49 @@ private String getUserId(HttpServletRequest request) throws IOException {
   }
 
   private String getUsername(String userId) {
-    Filter userIdFilter =
-        new FilterPredicate("userId", FilterOperator.EQUAL, userId);
+    Filter userIdFilter = new FilterPredicate("userId", FilterOperator.EQUAL, userId);
     Query userQuery = new Query("User").setFilter(userIdFilter);
     PreparedQuery result = datastore.prepare(userQuery);
     Entity currentUser = result.asSingleEntity();
     return (String) currentUser.getProperty("username");
   }
 
   private boolean checkIfUserPreviouslyGuessedCorrect(String userId, EmbeddedEntity currentRound) {
-    EmbeddedEntity userGuessStatuses = (EmbeddedEntity) currentRound.getProperty("userGuessStatuses");
+    EmbeddedEntity userGuessStatuses =
+        (EmbeddedEntity) currentRound.getProperty("userGuessStatuses");
     boolean userGuessStatus = (Boolean) userGuessStatuses.getProperty(userId);
     return userGuessStatus;
   }
 
-  private void markUserGuessedCorrectly(String userId, EmbeddedEntity currentRound, Entity currentGame) {
-    EmbeddedEntity userGuessStatuses = (EmbeddedEntity) currentRound.getProperty("userGuessStatuses");
+  private void markUserGuessedCorrectlyAndAddPoints(
+      String userId, EmbeddedEntity currentRound, Entity currentGame) {
+    // Changes the users "guess status" to show they've guessed correctly
+    EmbeddedEntity userGuessStatuses =
+        (EmbeddedEntity) currentRound.getProperty("userGuessStatuses");
     userGuessStatuses.setProperty(userId, true);
+    
+    // Increases the users points by a fixed amount
```

Comment:
We have plans to assign points based on how early a user guesses correctly, right (e.g. first correct guess gets most points, second correct guess gets second most points, etc)? If so, maybe add a TODO for this here